presto-main/src/main/java/com/facebook/presto/cost/StatisticRange.java

```diff
@@ -28,22 +38,40 @@
 
 public class StatisticRange
 {
-    private static final double INFINITE_TO_FINITE_RANGE_INTERSECT_OVERLAP_HEURISTIC_FACTOR = 0.25;
-    private static final double INFINITE_TO_INFINITE_RANGE_INTERSECT_OVERLAP_HEURISTIC_FACTOR = 0.5;
+    protected static final double INFINITE_TO_FINITE_RANGE_INTERSECT_OVERLAP_HEURISTIC_FACTOR = 0.25;
+    protected static final double INFINITE_TO_INFINITE_RANGE_INTERSECT_OVERLAP_HEURISTIC_FACTOR = 0.5;
+
+    /**
+     * Serialized length in bytes. 3 doubles + 2 booleans
+     */
+    private static final int SERIALIZED_LENGTH = 3 * Double.BYTES + 2;
```

Comment:
```suggestion
    private static final int SERIALIZED_LENGTH_IN_BYTES = 3 * Double.BYTES + 2;
```

Reply:
Given the changes to histograms to not serialize them when sending over plan fragments and potentially to not serialize them at all to workers, I've opted to revert the change to the serialization format of the the `StatisticRange` class.